src/fmu/tools/sensitivities/_designsummary.py

```diff
@@ -53,15 +53,17 @@ def summarize_design(filename, sheetname="DesignSheet01"):
 
     # Read design matrix and find realisation numbers for each sensitivity
     if filename.endswith(".xlsx"):
-        dgn = pd.read_excel(filename, sheetname)
+        dgn = pd.read_excel(filename, sheetname, engine="openpyxl")
+        dgn.dropna(axis=0, how="all", inplace=True)
```

Comment:
Is this due to things like "coloured cells"? If so, maybe nice to add a comment about that (to explain why the line of code is there, otherwise it could disappear in later refactorings)

Reply:
Have added this